pkgs/development/python-modules/django-widget-tweaks/default.nix

```diff
@@ -1,4 +1,16 @@
-{ buildPythonPackage, fetchFromGitHub, python, lib, django }:
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+
+# native
+, setuptools-scm
+
+# propagated
+, django
+
+# tests
```

Comment:
```suggestion
, fetchFromGitHub
, setuptools-scm
, django
```

Reply:
Are you really suggesting to *remove* code comments? Why?

Reply:
> Are you really suggesting to remove code comments? Why?

We are not removing any useful comments here, we are not adding unhelpful ones.
They only duplicate information which can already be easily looked at a few lines below and create new questions like what happens if something is used in native, propagatedBuildInputs and tests. Also they add new meta data which needs to be maintained and if bad maintained makes the code harder to understand.